### DISCUSSION TITLE
Syntax highlighting: SQL

### DISCUSSION THREAD

**Discussion by mharig** [Textualize/rich#2063] [Ideas] [answered]
Is it planned? How may I help implementing it?

**Comment by willmcgugan** [comment, marked as answer]
```python
from rich import print
from rich.syntax import Syntax
print(Syntax("SELECT * FROM Customers WHERE CustomerID=1;", lexer="mysql", theme="monokai"))
```

**Reply by mharig** [reply]
I see, thanks. What extension does rich-cli expect for a SQL file, then? It does not work with .sql .

**Reply by mharig** [reply]
After a quick look in the rich-cli source code, I found that it supports all that pygments support.
Syntax e.g.:
`rich --lexer sql  <myscript.sql>`
